fe/fe-core/src/main/java/com/starrocks/http/rest/TableQueryPlanAction.java

```diff
@@ -180,11 +178,11 @@ protected void executeWithoutPassword(BaseRequest request, BaseResponse response
             resultMap.put("status", e.getCode().code());
             resultMap.put("exception", e.getMessage());
         }
-        ObjectMapper mapper = new ObjectMapper();
```

Comment:
why remove?
`mapper` is used below.

Reply:
A static mapper instance has been defined in the parent class `RestBaseAction`, and i think it's better to reuse it.